packages/tools/devtools/devtools-core/src/data-visualization/SharedTreeVisualizer.ts

```diff
@@ -24,15 +28,66 @@ import {
 	type VisualTreeNode,
 } from "./VisualTree.js";
 
+/**
+ * Returns allowed types of the non-leaf nodes in the tree.
+ * - String: Allowed type for the array node.
```

Comment:
The docs for this seems a bit confusing to read. 
Shouldn't they follow this format

/**
 * Returns a VisualChildNode  of non-leaf nodes in the tree
 *
 * @param allowedTypes - a string if array node, Record<string, string>  for non-array nodes
 * @returns a VisualChildNode with the allowed type
 */